include/library/spdm_lib_config.h

```diff
@@ -19,12 +19,16 @@
 #define MAX_SPDM_CERT_CHAIN_BLOCK_LEN 1024
 
 #define MAX_SPDM_MESSAGE_BUFFER_SIZE 0x1200
-#define MAX_SPDM_MESSAGE_SMALL_BUFFER_SIZE 0x100
+#define MAX_SPDM_MESSAGE_SMALL_BUFFER_SIZE 0x100  // to hold message_a before negotiate
+#define MAX_SPDM_MESSAGE_MEDIUM_BUFFER_SIZE 0x300 // to hold message_k before finished_key is ready
 
 #define MAX_SPDM_REQUEST_RETRY_TIMES 3
 #define MAX_SPDM_SESSION_STATE_CALLBACK_NUM 4
 #define MAX_SPDM_CONNECTION_STATE_CALLBACK_NUM 4
 
+// If cache transcript data or transcript hash
+//#define RECORD_TRANSCRIPT_DATA 1

```

Comment:
nitpicky, but should we handle similarly to OPENSPDM_*_SUPPORT defines below?
Where support is enabled/disabled with 1 or 0, rather than "#ifdef"?

Reply:
that is OK.

I think we need clean up the *OPENSPDM* to *LIBSPDM*.

Reply:
agreed, just didn't want to clutter this CL with it.
as for 1&0 vs #define, mostly want to be consistent, rather than strong preference for either.

Reply:
Fixed